lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +22,12 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    text = text.lower()
+    lst = []
+    for letter in text:
```

Comment:
Too hard. Can you do this work with one loop?

Reply:
shortened